api/logic/tag_janitor_test.go

```diff
@@ -39,33 +48,17 @@ func TestTagJanitorPulse(t *testing.T) {
 
 	//expecting 3 tags for task1 to be deleted from the tagstore
 	testutils.AssertEqual(t, tagsAdded-3, len(tags))
-	testutils.AssertEqual(t, "task2", tags[0].EntityID)
+	testutils.AssertEqual(t, "t2", tags[0].EntityID)
 }
 
-func getTagStore() (tag_store.TagStore, int) {
+func getTagStore(tags []models.Tag) (tag_store.TagStore, int) {
```

Comment:
not sure why you need to return the `tasksAdded int` since the user knows this from the input param?